### ISSUE TITLE
Where to know how to print loss for new loss function when using TF-Slim?

### ISSUE THREAD

**Issue by stoneyang** [tensorflow/models#5559]
Hi, there:

You guys did a great job, no doubt. 

But the documents seems severely unfriendly to new comer, at least like me myself.

The caption's my problem, how simple, but I just can't find where to learn it....

Could anyone shed some light? Thanks in advance!

I want to train a model using `tf-slim`, in a minimal modification manner, ie, I just add **a custom loss function** and train a multi-loss model. The function's adapted to slim, running, but I just cannot print the loss values of it.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
